src/index.tsx

```diff
@@ -80,3 +87,73 @@ export const Zage = ({
     </Modal>
   );
 };
+
+export const ZageInfoModal = ({ publicKey, showModal, setShowModal }: ZageInfoModalProps) => {
+  const [jsResponse, setJsResponse] = useState<string>('');
+  let webview:any;
+  const getJsRes = async (publicKey: string) => {
+    try {
+      const req = new XMLHttpRequest();
+      req.onreadystatechange = () => {
+        const appendedJs =
+          req.responseText +
+          `
+          openModal('${publicKey}')
+          true
+        `;
+        setJsResponse(appendedJs);
+      };
+      req.open('GET', 'infoModalPlaceHolder', true); //Change URL to Info Modal URL
+      req.send(null);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+  useEffect(() => {
+    if (jsResponse === '') {
+      getJsRes(publicKey);
+    }
+  }, [publicKey, jsResponse]);
+  return (
+    <Modal
+      visible={showModal}
+      transparent={true}
+      presentationStyle="overFullScreen"
+      style={{
+        backgroundColor: "transparent",
+        borderTopLeftRadius: 20,
+        borderTopRightRadius: 20,
+      }}
+      animationType="slide"
+    >
+      <WebView
+        ref={(ref) => (webview = ref)}
+        useWebKit={true}
+        height={"100%"}
+        width={"100%"}
+        source={{ uri: INFO_MODAL_URL }}
+        javaScriptEnabled={true}
+        injectedJavaScript={jsResponse}
+        style={{
+          resizeMode: "contain",
+          backgroundColor: "white",
+          marginTop: 50,
+          borderColor: "black",
+          borderWidth: 5,
+          borderRadius: 40,
+        }}
+        onMessage={() => {
+          setShowModal(false);
+        }}
+        onNavigationStateChange={(_) => {
+          if (_.url !== INFO_MODAL_URL && Platform.OS === "ios") {
```

Comment:
so this isn't an issue on Android at all?

Reply:
No this isn't an issue for android. Based on what I learned in the react-native discord and testing it on the android.